Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,14 @@
 
 ## Exercise I
 
-a)
+a) O(n) ~ 0(n^3) divided by O(n^2) == O(n)
 
 
-b)
+b) O(n^2) ~ O(n) plus O(n) == O(n^2)
 
 
-c)
+c) O(n) ~ function would be dependent on bunnies in other words n
 
 ## Exercise II
-
+I would create a binary algorithm also known as a divide and conquer algorithm that starts by droping the egg from the middle of the list and if the egg bracks the algorithm will search the lower part of the list/floor if the egg does not brack the algorithm will search the greater part of the list/floor by comparing the list by twos until if finds the floor where the egg will brack. The runtime complexity would be O(n)
```

Comment:
Great explanation. Only thing to change would be your time complexity analysis. Binary search runs in O(log(n))